doc/manual/manual.tex

```diff
@@ -1315,14 +1316,233 @@ \subsection{Numerical methods}
+
+The \textit{Anelastic Liquid Approximation (ALA)} is based on two assumptions.
+First, that the density variations $\rho(p,T)-\bar\rho$ are small and in
+particular can be accurately described by a Taylor expansion:
+\begin{align*}
+  \rho(p,T) \approx 
+  \bar\rho 
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial p} p'
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial T} T'.
+\end{align*}
+Here, $\frac{\partial \rho(\bar p,\bar T)}{\partial T}$ is related to the
+thermal expansion coefficient, and $\frac{\partial \rho(\bar p,\bar T)}{\partial p}$ to the
+compressibility.
+
+The second assumotion is that the variation of the density
+from the reference density can be neglected in the mass balance and
+temperature equations.
+This yields the following system of equations for the velocity and pressure
+equations, using the definition of the adiabatic hydrostatic pressure field
+$\bar p$:
+\begin{align}
+  \label{eq:stokes-ALA-1}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+                \right] + \nabla p' &=
+  \left(\frac{\partial \rho(\bar p,\bar T)}{\partial p} p'
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial T} T' \right) \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-ALA-2}
+  \nabla \cdot (\bar\rho \mathbf u) &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+\marginpar{Timo: need to figure out how to solve \eqref{eq:stokes-ALA-2}, need $\nabla \rho$! }
+
+For the temperature equation we arrive at the following:
+\marginpar{Scott: This does not match (18) in \cite{KLKLZTTK10}}
+
+\begin{multline}
+  \label{eq:temperature-decomposed}
+  \bar\rho C_p \left(\frac{\partial T'}{\partial t} + \mathbf u\cdot\nabla
+  T'\right) - \nabla\cdot k\nabla T'
+  \\
+  =
+  \rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha T' \left( \mathbf u \cdot \nabla p \right)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+Timo: this is not quite correct, you need reference density and simplified adiabatic heating to get:
+\begin{multline}
+  \label{eq:temperature-decomposed2}
+  \bar\rho C_p \left(\frac{\partial T'}{\partial t} + \mathbf u\cdot\nabla
+  T'\right) - \nabla\cdot k\nabla T'
+  \\
+  =
+  \bar\rho H
+  +
+  2\eta
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  :
+  \left(\varepsilon(\mathbf u) - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+  +\alpha \bar\rho T' (\mathbf u \cdot \mathbf g)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+
+
+
+\subsubsection{The truncated anelastic liquid approximation (TALA)}
+
+The \textit{Truncated Anelastic Liquid Approximation (TALA)} further simplifies
+the ALA by assuming that the variation of the density due to pressure variations
```

Comment:
too many "variations" :-) Maybe say "that the variation of the density due to $p'$ is small"?